orm/nuxt/README.md

```
@@ -139,23 +145,23 @@ The app is now running, navigate to [`http://localhost:3000/`](http://localhost:
 
 <br />
 
-**Blog** (located in [`./pages/index.vue`](./pages/index.vue)
+**Blog** (located in [`./app/pages/index.vue`](./app/pages/index.vue)
 
 ![](https://imgur.com/eepbOUO.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)